integration-templates/xero/mappers/to-invoice.ts

```diff
@@ -100,6 +116,14 @@ export function toXeroInvoice(invoice: CreateInvoice | Invoice) {
             xeroItem['TaxAmount'] = item.taxes_amount_cents / 100;
         }
 
+        if (item.discount_amount_cents) {
+            xeroItem['DiscountAmount'] = item.discount_amount_cents / 100;
```

Comment:
manipulating money with arithmetic operations can lead to rounding issue. Could it be the case here?

Reply:
Made some adjustments to be a bit more precise https://github.com/NangoHQ/nango/pull/2519/commits/287742b4efc37af3b0b6e02abe20c8af2bd53e69. Worth noting that this logic was taken from an existing client you're familiar with that is in production